aeat_mapping.py

```diff
@@ -233,6 +247,31 @@ def _build_issuer_id(self, invoice):
             'NIF': self.nif(invoice),
         }
 
+    def build_taxes(self, tax):
+        res = {
+            'TipoImpositivo': tools._rate_to_percent(self.tax_rate(tax)),
+            'BaseImponible': self.tax_base(tax),
+            'CuotaRepercutida': self.tax_amount(tax)
+            }
+
+        if self.tax_equivalence_surcharge_rate(tax):
+            res['TipoRecargoEquivalencia'] = (
+                tools._rate_to_percent(self.tax_equivalence_surcharge_rate(
+                        tax)))
+
+        if self.tax_equivalence_surcharge_amount(tax):
+            res['CuotaRecargoEquivalencia'] = (
+                self.tax_equivalence_surcharge_amount(tax))
+        return res
+
+    def location_rules(self, invoice):
+        base = 0
+        for line in invoice.lines:
+            for tax in line.taxes:
+                if tax.sii_issued_key == '08':
+                    base += attrgetter('amount')(line)
```

Comment:
It seems to me that there should be a "break" after "base += ....". After all, we don't want to sum the amount twice if there are two taxes with "08".

Though I understand this should never happen...

Reply:
> It seems to me that there should be a "break" after "base += ....". After all, we don't want to sum the amount twice if there are two taxes with "08".
> 
> Though I understand this should never happen...

The same as in the not_subject function. We need to ensure that there aren't more taxes defined identically.